examples/06-patchman.yaml

```diff
@@ -31,6 +35,7 @@ hosts:
     on-package-updates:
     - apt:
         autoremove: '{{ autoremove }}'
+        withnewpkgs: '{{ withnewpkgs }}'
```

Comment:
also here. It is fine to keep the parameter as `withnewpkgs`

Reply:
done